packages/teleterm/src/ui/services/usageEvent/usageEventService.ts

```diff
@@ -17,16 +17,21 @@
 import { ClusterOrResourceUri, ClusterUri, routing } from 'teleterm/ui/uri';
 import {
   Cluster,
-  ReportEventRequest,
+  UsageEventWithDate,
   TshClient,
 } from 'teleterm/services/tshd/types';
 import { RuntimeSettings } from 'teleterm/mainProcess/types';
 import { ConfigService } from 'teleterm/services/config';
 import Logger from 'teleterm/logger';
 
-const logger = new Logger('UsageEventService');
+type RawPrehogEvent = Omit<
+  UsageEventWithDate['prehogEvent'],
+  'distinctId' | 'timestamp'
+>;
 
 export class UsageEventService {
```

Comment:
What do you think about renaming this to `UsageService` and then we could have `reportEvent` instead of `reportUsageEvent`? I'm thinking of Go best practices where they talk about avoiding repetitions in package & struct names.

Reply:
Sure, done